sdk/tests/webgl-conformance-tests.html

```diff
@@ -257,7 +258,10 @@
     }
     div.appendChild(button);
     var a = reporter.localDoc.createElement('a');
-    a.href = WebGLTestHarnessModule.getURLWithOptions(url, reporter.selectedWebGLVersion);
+    a.href = WebGLTestHarnessModule.getURLWithOptions(url, {
+      webglVersion: reporter.selectedWebGLVersion,
+      quiet: OPTIONS.quiet
```

Comment:
Before committing this, could you please go through the exercise of wiring this up to Chromium's webgl_conformance.py harness and making sure that the quiet mode can be easily specified? I think using query arguments in the URL may cause problems -- possibly with the test expectations mechanism. Note that webgl_conformance.py overrides portions of the code in sdk/tests/js/webgl-test-harness.js , specifically reportResults -- if it's possible to inject the "quiet" variable with code, that may be a more robust solution.